static/app/components/u2f/u2fContainer.tsx

```diff
@@ -1,43 +1,19 @@
 import {Component} from 'react';
 
-import {Client} from 'sentry/api';
 import {Authenticator} from 'sentry/types';
-import withApi from 'sentry/utils/withApi';
 
 import U2fSign from './u2fsign';
 
 type Props = {
-  api: Client;
+  authenticators: Array<Authenticator>;
   onTap: U2fSign['props']['onTap'];
   className?: string;
   displayMode?: U2fSign['props']['displayMode'];
 };
-type State = {
-  authenticators: Array<Authenticator>;
-};
-
-class U2fContainer extends Component<Props, State> {
-  state: State = {
-    authenticators: [],
-  };
-  componentDidMount() {
-    this.getAuthenticators();
-  }
-
-  async getAuthenticators() {
-    const {api} = this.props;
-
-    try {
-      const authenticators = await api.requestPromise('/authenticators/');
-      this.setState({authenticators: authenticators ?? []});
-    } catch {
-      // ignore errors
-    }
-  }
 
+class U2fContainer extends Component<Props> {
```

Comment:
hi @priscilawebdev 
do you have any opinions on removing this component?

Richards now passing the authenticators response from the parent component down to this container (which just passes it to its children)n

Seems like this is now unnecessary and we can merge this with the SudoModal.
I'm not familiar with the patterns we may be attempting to encourage for FE file structure though.